driver/src/price_estimation/dexag/api.rs

```diff
@@ -0,0 +1,167 @@
+#![allow(dead_code)]
+
+use anyhow::{anyhow, Context, Result};
+use ethcontract::Address;
+use isahc::prelude::*;
```

Comment:
I'm not a huge fan of * imports. I think it's better to be explicit about what aspects we need.

Reply:
I usually agree but prelude modules are usually designed to be *-imported this way. I’m ok with ether in this case.